faces-tck/pom.xml

```diff
@@ -0,0 +1,162 @@
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License v. 2.0, which is available at
+    http://www.eclipse.org/legal/epl-2.0.
+
+    This Source Code may also be made available under the following Secondary
+    Licenses when the conditions for such availability set forth in the
+    Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+    version 2 with the GNU Classpath Exception, which is available at
+    https://www.gnu.org/software/classpath/license.html.
+
+    SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <artifactId>tck</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>payara-external-tck-faces</artifactId>
+    <name>TCK: Faces</name>
+
+    <description>This verifies the compliance of Payara using the Jakarta Faces standalone TCK</description>
+
+    <properties>
+        <faces.version>4.1.1</faces.version>
+        <tck.root>${project.build.directory}</tck.root>
+        <sigtest.api.version>4.1.1</sigtest.api.version> <!-- version of org.glassfish:jakarta.faces used in Payara -->
+        <maven.executable>${maven.home}/bin/mvn</maven.executable>
+        <maven.settings.xml>${user.home}/.m2/settings.xml</maven.settings.xml>
+        <maven.exec.params>-B -e -s ${maven.settings.xml} -f ${tck.root}/faces-tck-${faces.version}/tck/pom.xml</maven.exec.params>
+        <maven.exec.projects></maven.exec.projects>
+        <maven.exec.goals>clean install</maven.exec.goals>
+        <maven.exec.basic.options>-Ppayara-ci-remote -P!glassfish-ci-managed -Dpayara.version=${payara.version} -Dfaces.version=${faces.version} -Dsigtest.api.groupId=org.glassfish -Dsigtest.api.artifactId=jakarta.faces -Dsigtest.api.version=${sigtest.api.version}</maven.exec.basic.options>
+        <maven.exec.options></maven.exec.options>
+        
+        <!--<maven.exec.skipping>-Dtest.selenium=false -pl !org.eclipse.ee4j.tck.faces.old-tck-selenium:ajax,!org.eclipse.ee4j.tck.faces.old-tck-selenium:commandLink,!org.eclipse.ee4j.tck.faces.old-tck-selenium:protectedViews</maven.exec.skipping>--> <!-- these options should be removed for the final version -->
+        <!-- if doing selenium tests (-Dtest.selenium=true) and your system Chrome is old, use -Dchrome.binary=/path/to/your/chrome -->
+        <maven.exec.skipping>-Dtest.selenium=false -Dtck.old.skip=true -pl !:old-faces-tck-parent,!:old-tck-build,!:old-tck-run,!org.eclipse.ee4j.tck.faces.old-tck-selenium:ajax,!org.eclipse.ee4j.tck.faces.old-tck-selenium:ajax,!:commandLink,!:protectedViews</maven.exec.skipping>
```

Comment:
I wouldn't skip the old TCK.
We're not interested in having Jenkins pass at the price of skipping tests - we should be able to use Jenkins to test if we've got a working TCK runner for Payara without having to adjust this config again.